src/Runner.Worker/ActionManifestManager.cs

```diff
@@ -104,18 +118,18 @@ public ActionDefinitionData Load(IExecutionContext executionContext, string mani
             catch (Exception ex)
             {
                 Trace.Error(ex);
-                context.Errors.Add(ex);
+                templateContext.Errors.Add(ex);
             }
 
-            if (context.Errors.Count > 0)
+            if (templateContext.Errors.Count > 0)
             {
-                foreach (var error in context.Errors)
+                foreach (var error in templateContext.Errors)
                 {
                     Trace.Error($"Action.yml load error: {error.Message}");
                     executionContext.Error(error.Message);
                 }
 
-                throw new ArgumentException($"Fail to load {manifestFile}");
+                throw new ArgumentException($"Fail to load {fileName}");
```

Comment:
will this always shows `actions.yml`?

Reply:
TODO for ethan: Change variable name top a more clear name

Reply:
discussed offline, it's the relative path. Would it help to name the variable `relative___`?